larpandoracontent/LArObjects/LArCaloHit.h

```diff
@@ -28,6 +28,7 @@ class LArCaloHitParameters : public object_creation::CaloHit::Parameters
 {
 public:
     pandora::InputUInt      m_larTPCVolumeId;       ///< The lar tpc volume id
+    pandora::InputUInt      m_daughterVolumeId;     ///< The daughter volume id
```

Comment:
These changes appear to be a subset of the changes in Maria's branch. As with the master algorithm, it looks like the simple option here is to pick Maria's changes for this file.